testem.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
                 '--remote-debugging-port=9222',
                 '--window-size=1440,900'
             ]
-        }
+        },
+        "parallel": -1
```

Comment:
why `-1`? shouldn't this also be `4` like in the `ember exam` call?

Reply:
This will default to the max number of browsers needed.
If we change ember exam to 8, 16, 32, it will run the same number of browser.

Reply:
hmm, if you look at the TravisCI output it still looks like the test chunks are running sequentially though and not in parallel 🤔